nx.json

```diff
@@ -30,6 +30,9 @@
   "targetDefaults": {
     "build": {
       "dependsOn": ["^build"]
+    },
+    "test": {
+      "outputs": ["packages/*/coverage"]
```

Comment:
It would be nice if we could specify something like `["{projectBeingRunOn}/coverage"]`, but I couldn't find nx docs describing how to do such a thing.

Reply:
@JoshuaKGoldberg that’s almost exactly it - it’s {projectRoot}

Reply:
You can see more info here https://nx.dev/concepts/how-caching-works

Reply:
Aha I mistook that to be a path like `~/repos/typescript-eslint`, not `~/repos/typescript-eslint-package/type-utils`. Awesome!

Reply:
Aha I mistook that to be a path like `~/repos/typescript-eslint`, not `~/repos/typescript-eslint-package/type-utils`. Fantastic. Thanks!

Reply:
```suggestion
      "outputs": ["{projectRoot}/coverage"]
```